yb-voyager/cmd/common.go

```diff
@@ -578,3 +578,14 @@ func GetSourceDBTypeFromMSR() string {
 	}
 	return msr.SourceDBConf.DBType
 }
+
+func hideFlagsInFallForwardOrBackCmds(cmd *cobra.Command) {
+	var flags = []string{"target-db-type", "import-type", "source-db-type", "export-type", "target-endpoints", "use-public-ip", "continue-on-error", "table-list",
```

Comment:
fall-forward setup command can have table-list, no? Don't see a problem with that..

Reply:
pls ensure table-list flags are not hidden from fall-back/forward synchronize

Reply:
Yes, we use different function for synchronize i.e. Export... one, so not removing from there.

Reply:
yes done.